gem/lib/pupperware/spec_helper.rb

```diff
@@ -41,13 +41,13 @@ def run_command(env = ENV.to_h, command, stream: STDOUT)
     Open3.popen3(env, command) do |stdin, stdout, stderr, wait_thread|
       stdout_reader = Thread.new do
         Thread.current.report_on_exception = false
-        stdout.each { |l| stdout_string << l and stream.puts l }
+        stdout.each { |l| stdout_string << l and stream.puts l and stream.flush() }
```

Comment:
I'm not even sure or convinced that this solves the problem... but I also don't think it hurts.